docs/examples/irradiance-transposition/use_perez_modelchain.py

```diff
@@ -0,0 +1,113 @@
+from pvlib import iotools, location, irradiance
+import pvlib
+import os
+import matplotlib.pyplot as plt
+
+# load in TMY weather data from North Carolina included with pvlib
+PVLIB_DIR = pvlib.__path__[0]
+DATA_FILE = os.path.join(PVLIB_DIR, 'data', '723170TYA.CSV')
+
+tmy, metadata = iotools.read_tmy3(DATA_FILE, coerce_year=1990,
+                                  map_variables=True)
+
+weather_data = pd.DataFrame({'ghi': tmy['ghi'], 'dhi': tmy['dhi'],
+                             'dni': tmy['dni'],
+                             'temp_air': tmy['temp_air'],
+                             'wind_speed': tmy['wind_speed']})
+
+loc = location.Location.from_tmy(metadata)
+
+surface_tilt = metadata['latitude']
+surface_azimuth = 180
+
+# define an example module and inverter
+sandia_modules = pvlib.pvsystem.retrieve_sam('SandiaMod')
+cec_inverters = pvlib.pvsystem.retrieve_sam('cecinverter')
+sandia_module = sandia_modules['Canadian_Solar_CS5P_220M___2009_']
+cec_inverter = cec_inverters['ABB__MICRO_0_25_I_OUTD_US_208__208V_']
+
+temp_params = TEMPERATURE_MODEL_PARAMETERS['sapm']['open_rack_glass_glass']
+
+# define the system and ModelChain
+system = PVSystem(arrays=None,
+                  surface_tilt=surface_tilt,
+                  surface_azimuth=surface_azimuth,
+                  module_parameters=sandia_module,
+                  inverter_parameters=cec_inverter,
+                  temperature_model_parameters=temp_params)
+
+mc = ModelChain(system, location=loc)
+
+# Cape Canaveral seems like the most likely match for climate
+model_perez = 'capecanaveral1988'
+
+solar_position = loc.get_solarposition(times=weather_data.index)
+dni_extra = irradiance.get_extra_radiation(weather_data.index)
+
+POA_irradiance = irradiance.get_total_irradiance(
+    surface_tilt=surface_tilt,
+    surface_azimuth=surface_azimuth,
+    dni=weather_data['dni'],
+    ghi=weather_data['ghi'],
+    dhi=weather_data['dhi'],
+    solar_zenith=solar_position['apparent_zenith'],
+    solar_azimuth=solar_position['azimuth'],
+    model='perez',
+    dni_extra=dni_extra)
+
+POA_irradiance_new_perez = irradiance.get_total_irradiance(
+    surface_tilt=surface_tilt,
+    surface_azimuth=surface_azimuth,
+    dni=weather_data['dni'],
+    ghi=weather_data['ghi'],
+    dhi=weather_data['dhi'],
+    solar_zenith=solar_position['apparent_zenith'],
+    solar_azimuth=solar_position['azimuth'],
+    model='perez',
+    model_perez=model_perez,
+    dni_extra=dni_extra)
+
+mc.run_model_from_poa(POA_irradiance)
+ac_power_default = mc.results.ac
+
+mc.run_model_from_poa(POA_irradiance_new_perez)
+ac_power_new_perez = mc.results.ac
+
+start, stop = '1990-05-05 06:00:00', '1990-05-05 19:00:00'
+plt.plot(ac_power_default.loc[start:stop],
+         label="Default Composite Perez Model")
+plt.plot(ac_power_new_perez.loc[start:stop],
+         label="Cape Canaveral Perez Model")
+plt.xticks(rotation=90)
+plt.ylabel("AC Power ($W$)")
+plt.legend()
```

Comment:
The bottom of the plot is getting cut off, consider using `plt.tight_layout()` or similar

Reply:
Worked on my setup, but I added `plt.tight_layout()` and hopefully it'll work now

Reply:
Ok, maybe matplotlib was just choosing a tick label formatting that I wasn't expecting.